app/components/Dashboard.jsx

```diff
@@ -19,11 +18,13 @@ const mapStateToProps = (state) => ({
   consumerPendingDuration: state.consumer.pendingDuration,
 });
 
+// Mapping dispatch actions to trigger state changes for Producer and Consumer metrics
```

Comment:
-make producer and consumer lowercase. we should be consistent, and in most places we have it lowercase
-you say we're mapping these actions... but to what? you should specify that they're being mapped to our props object